web/tests/functional/diff_local/test_diff_local.py

```diff
@@ -222,7 +222,6 @@ def test_suppress_reports(self):
         source code comments.
         """
         cfg = dict(self._codechecker_cfg)
-        cfg['analyzers'] = ['clang-tidy']
```

Comment:
Needed to remove this, because the codechecker call beneath enables clangsa checkers as well, which would cause an error.